README.md

```diff
@@ -1 +1,71 @@
-# capstone-project-1
\ No newline at end of file
+# capstone-project-1
+
+> capstone-project-1
+
+## Built With
+
+- HTML
+- CSS
+- javaScript
+
+## Design
+
+- Credit goes to Cindy Shin's design on [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
+    This is a mockup of Cindy's design.
+    
+
+## online version
+
+to see the online version [click here](https://boo-shehab.github.io/capstone-project-1/)
+
+## Getting Started
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- text editor
+
+- nodejs installed on your local machine
+
+- Cindy Shin's design on [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
+
+### Setup
+
+- Clone the repository
+
+- Open the project in your favourite text editor
+
+### Install
+
+git clone https://github.com/boo-shehab/capstone-project-1.git
+
+
+## 👤 Author1
+
+- GitHub: [@boo-shehab](https://github.com/githubhandle)
```

Comment:
Great job @boo-shehab 🤝

- [ ] [Optional] Kindly add your GitHub handle so that your Readme file can look more professional.